server/routes/flag.js

```diff
@@ -10,7 +10,7 @@ var connectedId;
 
 const io = require('socket.io')(PORT, {
     cors: {
-        origin: ['http://localhost:3000'] // Client URL -- http://localhost:3000 for testing
```

Comment:
Might need to add FRONTEND_URL="" in .env for server.
Then call it with process.env.FRONTEND_URL here

Reply:
OMG Ignore me, sorry.

Can you change this back to
const corsOptions = {
  origin: ['http://localhost:3000', 'https://sentinel-frontend.vercel.app'],
};

T_T